spinalcordtoolbox/reports/assets/index.html

```diff
@@ -262,7 +268,55 @@
         }
     }
 
-    <!-- Toggle overlay -->
+    function download_yaml() {
+        let res = {};
+        let seg_cmd_to_seg_type = {
+            "sct_analyze_lesion": "FILES_GMSEG",
+            "sct_analyze_texture": "FILES_SEG",
+            "sct_create_mask": "FILES_SEG",
+            "sct_crop_image": "FILES_SEG",
+            "sct_deepseg": "FILES_SEG",
+            "sct_deepseg_gm": "FILES_GMSEG",
+            "sct_deepseg_lesion": "FILES_GMSEG",
+            "sct_deepseg_sc": "FILES_SEG",
+            "sct_label_utils": "FILES_LABEL",
+            "sct_label_vertebrae": "FILES_LABEL",
+            "sct_process_segmentation": "FILES_SEG",
+            "sct_propseg": "FILES_SEG",
+            "sct_register_multimodal": "FILES_SEG",
+            "sct_register_to_template": "FILES_SEG",
+            "sct_resample": "FILES_SEG",
+            "sct_run_batch": "FILES_SEG",
+            "sct_smooth_spinalcord": "FILES_SEG",
+            "sct_straighten_spinalcord": "FILES_SEG",
+            "sct_warp_template": "FILES_SEG",
+        }
+        sct_data.forEach(item => {
+            if (item.qc === true) {
+                let command = item.command;
+                if (command in seg_cmd_to_seg_type) {
+                    segtype = seg_cmd_to_seg_type[command];
+                } else {
+                    segtype = "FIXME";
```

Comment:
TODO

Reply:
I've updated this to the following:

https://github.com/neuropoly/spinalcordtoolbox/blob/37fe602eaa0eceedee309c1d053cf47bccd49e9b/spinalcordtoolbox/reports/assets/index.html#L301

It's not the prettiest, but we should only hit this case if a new QC command is added and `seg_cmd_to_seg_type` isn't updated, so making it stand out is probably a good thing.

Writing that out, though, `seg_cmd_to_seg_type` feels error-prone; contributors have to remember to update an html doc when writing Python QC code, which I have a feeling we'll miss one day in the future. I wonder if it's possible to store the seg type in a column, and associate it with the Python code instead.